iib/web/exception_handlers.py

```diff
@@ -0,0 +1,37 @@
+import functools
```

Comment:
Could move this code to `iib/web/errors.py`? That Python module seems like a good fit for this.